pwiz_tools/Skyline/TestPerf/DiaSwathTutorialTest.cs

```diff
@@ -1116,74 +1134,84 @@ protected override void DoTest()
                     RunUI(() => changeGroupComparisonSettings.RadioScopePerPeptide.Checked = true);
                     OkDialog(changeGroupComparisonSettings, changeGroupComparisonSettings.Close);
 
-                    RunUI(() =>
-                    {
-                        var fcFloatingWindow = fcGrid.Parent.Parent;
-                        fcFloatingWindow.Left = SkylineWindow.Left + 8;
-                        fcFloatingWindow.Top = SkylineWindow.Bottom - fcFloatingWindow.Height - 8;
-                    });
+                    RunUI(() =>
+                    {
+                        var fcFloatingWindow = fcGrid.Parent.Parent;
+                        fcFloatingWindow.Left = SkylineWindow.Left + 8;
+                        fcFloatingWindow.Top = SkylineWindow.Bottom - fcFloatingWindow.Height - 8;
+                    });
+
+                    if (IsPasef)
+                    {
+                        ClickChromatogram(SkylineWindow.Document.MeasuredResults.Chromatograms[0].Name,
```

Comment:
You might consider copying the try/catch from line 864 to better handle the case when/if these hardcoded coordinates stop working. It's nicer behavior than just having the test abruptly stop with no chance to pick new coordinates.